apps/frontend/src/components/cards/SeverityChart.vue

```diff
@@ -14,7 +14,7 @@ import {Severity} from 'inspecjs';
 import Vue from 'vue';
```

Comment:
This sample will probably be replaced by the samples coming from the checklist mapper pr

Reply:
Moved this comment to the sample file diff